app/models/food.rb

```diff
@@ -0,0 +1,3 @@
+class Food < ApplicationRecord
```

Comment:
I am not really sure that food is a good name for describe this model. It like a Dish or so, but not food